templates/partials/greeting.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if recipientName ~}}
+<p>Hi {{recipientName}},</p>
+{{else ~}}
+<p>Hi,</p>
+{{/if ~}}
```

Comment:
Seeing this in context now, I realize it will add unnecessary markup in many places and we'll end up with things like `<h2><p>name</p></h2>`. I wonder if we shouldn't be unopinionated here and just go with:

```suggestion
{{#if recipientName ~}}
Hi {{recipientName}},
{{else ~}}
Hi,
{{/if ~}}
```

@nosro what do you think?